packages/react/src/input/input.styles.ts

```diff
@@ -567,7 +566,8 @@ export const StyledInputBlockLabel = styled('label', {
       underlined: true,
       asPlaceholder: true,
       css: {
-        left: '$space$2'
+        left: '$space$2',
+        top: '20%'
```

Comment:
The `labelPlaceholder` is still a bit out of position (vertically) when it is `undelined`.

![pr602-underline-label-placeholder](https://user-images.githubusercontent.com/32772271/178970433-83bef073-8237-47e0-a8a6-41c42d20b655.png)

Reply:
Do you mean they all need to be centered?

Reply:
Yep, it looks like all the Inputs that use `labelPlaceholder` have a shift of around 5 pixels up.

Hi @jrgarciadev is that a feature? 🤔

Reply:
Hey guys sorry for the delay, that was done on purpose due to the top animation